guides/php_upgrade.md

```diff
@@ -33,33 +34,36 @@ composer self-update --2
 
 :::: tabs
 ::: tab "NGINX"
-After upgrading to PHP 8.0, you will most likely need to update your NGINX configuration. Your configuration file
+After upgrading to PHP 8.1, you will most likely need to update your NGINX configuration. Your configuration file
 is most likely called `pterodactyl.conf` and located in the `/etc/nginx/sites-available/` directory, or if on CentOS,
 `/etc/nginx/conf.d/`.
 
 Make sure to update the path in the command below to reflect the actual location of your configuration file.
+
 ``` bash
-sed -i -e 's/php7.[0-4]-fpm.sock/php8.0-fpm.sock/' /etc/nginx/sites-available/pterodactyl.conf
+sed -i -e 's/php7.[0-4]-fpm.sock/php8.1-fpm.sock/' /etc/nginx/sites-available/pterodactyl.conf
 ```
 
 Once you have edited the file run the command below to reload nginx and apply your changes.
+
 ```bash
 systemctl reload nginx
 ```
 
 :::
 ::: tab "Apache"
-Run the commands below to disable PHP 7.4 and enable PHP 8.0 when serving requests. If you are upgrading from
+Run the commands below to disable PHP 7.4 and enable PHP 8.1 when serving requests. If you are upgrading from
 PHP 7.4 change the value in the command below to reflect that.
 
 ``` bash
 # Hint: a2enmod = a2_enable_module 🤯
-a2enmod php8.0
+a2enmod php8.1
 
 # Hint: a2dismod = a2_disable_module 🤯
 a2dismod php7.3
```

Comment:
Disable 7.3 upgrading from 7.4? We should cover 8 as well?

Reply:
Changed to a wildcard, so we'll disable any PHP versions users might have without guessing what it might be before enabling 8.1